layers.go

```diff
@@ -239,6 +239,9 @@ type LayerStore interface {
 	// ApplyDiff reads a tarstream which was created by a previous call to Diff and
 	// applies its changes to a specified layer.
 	ApplyDiff(to string, diff io.Reader) (int64, error)
+
+	// Cleanup re-loads the store and cleans invalid layers
+	Cleanup() error
```

Comment:
I don't like to name it `Cleanup`. It's effectively a wrapper around `Load` but locked. `LoadLocked` feels more accurate to me. The comment should reflect that it's a wrapper around Load.

Reply:
Sure, I fixed it and adapted as suggested.